### DISCUSSION TITLE
Project challenge with PDFs again

### DISCUSSION THREAD

**Discussion by Skye404** [USU-ITLS-x265-23-summer/class-repo#231] [Q&A] [answered]
I'm not sure why my PDFs and images aren't working. I've double checked the code. Can someone check it out, and let me know if you see something in error?

https://github.com/USU-ITLS-x265-23-summer/Skye404Project/tree/main

**Comment by jakegros** [comment, marked as answer]
That is frustrating for sure.
If they are files in your assets/css location, just double check that you are following the path: 
src="assets/css/exact file name here"

**Reply by Skye404** [reply]
Hmm, I double checked some things for the 100th time, and it finally worked. Thanks! I'm using this code though: href="assets/css/exact file name here"

**Comment by derekscholes** [comment]
Did you resolve the issue with the PDFs? Were you just wanting to display them or to have them available for others to download directly from the page?

**Reply by Skye404** [reply]
I finally got the other PDFs to work. They open in another tab. It's working well now. Thanks for checking.
